matlab/lib/+Ice/PropertyException.m

```diff
@@ -1,9 +1,6 @@
-% PropertyException   Summary of PropertyException
-%
-% This exception is raised when there is an error while getting or setting a property. For example, when
-% trying to set an unknown Ice property.
-
-% Copyright (c) ZeroC, Inc.
-
 classdef (Sealed) PropertyException < Ice.LocalException
+    %PROPERTYEXCEPTION The exception that is thrown when a property cannot be set or retrieved. For example, this
+    %   exception is thrown when attempting to set an unknown Ice property.
```

Comment:
PropertyException. We can also throw this for when a property has an invalid value.

Reply:
This description was copied from the C++ description.

Reply:
It should be fixed there as well!